tests/compile/test_config.py

```diff
@@ -26,15 +26,17 @@ def test_use_cudagraphs_dynamic(monkeypatch):
     assert not vllm_config.compilation_config.use_cudagraph
 
 
+# bug: vLLM doesn't free all memory on deletion so run in different process
```

Comment:
Could we make this an issue? It was resolved at some point and it must have been broken again. Then at least hopefully someone adds a test to make sure this doesn't happen again.

Reply:
Yup, https://github.com/vllm-project/vllm/issues/21073